packages/blade/src/components/TextArea/_decisions/decisions.md

```diff
@@ -0,0 +1,70 @@
+# TextArea
+
+A textarea field is an input field that allow users to input custom text entries which spans over multiple lines with a keyboard.
+
+<img src="./textarea-thumbnail.png" width="380" alt="" />
+
+## Design
+
+- [Figma - Alerts](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=11115%3A166804)
+
+## TextArea API
+
+**Sample Usage**
+
+```tsx
+<TextArea
+  label="Enter Message"
+  labelPosition="left"
+  placeholder="Enter your message"
+  showClearButton
+  helpText="Send message to author"
+  errorText="Message cannot be blank"
+  successText="Message sent"
+  validationState={textareaValue.length < 10 ? 'error' : 'success'}
```

Comment:
can we have a the warning state also for text area ?

Reply:
Design doesn't have warning state for any inputs. 
https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=11115%3A166743